internal/adapters/scheduler_storage/pg/scheduler_version.go

```diff
@@ -29,12 +29,14 @@ import (
 
 type SchedulerVersion struct {
 	Version   string      `db:"version"`
+	IsActive  bool        `db:"is_active"`
```

Comment:
@guilhermocc IsActive(is_active) is just a logical property, so we won't have a column in the database to represent it, right?

Reply:
exactly! in the query, we use an alias `is_active` for this calculated query result, so I had to add this annotation here